File: Program59.cpp
#include<iostream>
using namespace std;

class Nnumbers
{
    private:
        int Arr[5];
    public:
       void Accept()
       {
           int iCnt = 0;
           cout<<"Enter elements:"<<endl;
           for(iCnt = 0; iCnt < 5; iCnt++)
           {
               cin>>Arr[iCnt];
           }
       }
       void Display()
       {
           int iCnt = 0;
           cout<<"Elements of array are:"<<endl;
           for(iCnt = 0; iCnt < 5; iCnt++)
           {
               cout<<Arr[iCnt]<<endl;
           }
       }
};

int main()
{
    Nnumbers nobj;

    nobj.Accept();
    nobj.Display();
    
    return 0;
}

File: Program59.java
import java.lang.*;
import java.util.*;

class ArrayX
{
   private int Arr[];

   public void Accept()
   {
       int iSize = 0, iCnt = 0;

       Scanner sobj = new Scanner(System.in);
       System.out.println("Enter number of elements");
       iSize = sobj.nextInt();

       Arr = new int[iSize];

       System.out.println("Enter elements of Array:");
       
       for(iCnt = 0; iCnt < Arr.length; iCnt++)
       {
           Arr[iCnt] = sobj.nextInt();
       }
   }
       
   public void Display()
   {
    int iCnt = 0;   
    System.out.println("Elements of Array are:");
       
       for(iCnt = 0; iCnt < Arr.length; iCnt++)
       {
           System.out.println(Arr[iCnt]);
       }

   }
}

class Program59
{
    public static void main(String arg[])
    {
       ArrayX nobj = new ArrayX();

       nobj.Accept();
       nobj.Display();

    }
}